src/ifElse.test.js

```diff
@@ -1,11 +1,53 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(ifElse).toBeInstanceOf(Function);
+  });
+
+  it('should run a first callback if condition returns true', () => {
```

Comment:
Use a template literal for consistency with other test descriptions, e.g., `should run the first callback if the condition returns true`